VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -191,4 +194,27 @@ public static void setEndorsedState(@NonNull TextView target,
         }
 
     }
+
+    /**
+     * Renders hyperlinks in the given html to clickable items when applied to the given TextView.
```

Comment:
> and sets the text on the text view as well

This is still not done in 3e7117311b7dbb4fa2b223ed94871429b01ac336. You should probably also update it in the `textView` parameter documentation, e.g. "The text view **on which to apply** the provided HTML".